### DISCUSSION TITLE
Blog comment on Better automated testing with Embedded Weaviate

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1028] [Docs/blog comments]
# Blog comment on Better automated testing with Embedded Weaviate

Learn how to make testing less of a chore with Embedded Weaviate, and other tips for better automated testing.

https://weaviate.io/blog/automated-testing?utm_source=newsletter.weaviate.io&utm_medium=newsletter&utm_campaign=we-love-streamlit-datadog-loves-weaviate-and-more

**Comment by weisisheng** [comment]
Outstanding walkthrough.  Thanks!
